pkgs/development/ocaml-modules/thread-table/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchurl
+, buildDunePackage
+, mdx
+}:
+
+buildDunePackage rec {
+  pname = "thread-table";
+  version = "0.1.0";
+
+  minimalOCamlVersion = "4.08";
+  duneVersion = "3";
+
+  src = fetchurl {
+    url = "https://github.com/ocaml-multicore/${pname}/releases/download/${version}/${pname}-${version}.tbz";
+    sha256 = "d84BwC9W5udWJgYuaQwmA1e2d6uk0v210M7nK72VjXs=";
+  };
+
+  checkInputs = [
+    mdx
+  ];
```

Comment:
The `alcotest` library is also needed.

Btw, you must set the `doCheck` attribute in order to enable the tests.

Reply:
Great catch! I didn’t have it in `domain-local-await` either (I ran `cp domain-local-await/default.nix thread-table/`).